recipe/meta.yaml

```diff
@@ -11,33 +11,43 @@ source:
   sha256: 288509d28ed062ddf8b08212f65afb3aa07762a9a84bc5da9f7132016c8dd31f
 
 build:
-  number: 0
-  skip: True  # [win or osx]
+  number: 2
+  skip: True  # [osx or (win and py36)]
   features:
-    - blas_{{ variant }}
+    - blas_{{ variant }}  # [not win]
+    - vc9  # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and (py35 or py36)]
 
 requirements:
   build:
     - cmake
+    - python  # [win]
     - fftw
     - gdal 2.2.*
-    - ghostscript
+    - ghostscript  # [not win]
```

Comment:
`ghostscript` seems to provide some `.sln` files but I don't think those cover the `msvc` versions we need :unamused: 

Would a "re-packaging" of the binary blob work for `GMT`?